executor/executor.go

```diff
@@ -475,6 +501,9 @@ func (e *ReverseExec) Close() error {
 }
 
 func init() {
+	// While doing optimization in plan package, we need to execute non-correlated subquery,
+	// but plan package can not import executor package because of dependency cycle.
```

Comment:
-> but the plan package cannot import the executor package because of the dependency cycle.